src/zenml/cli/secret.py

```diff
@@ -215,9 +218,12 @@ def register_secret(
             while True:
                 k = click.prompt("Please enter a secret-key")
                 if k not in secret_contents:
-                    secret_contents[k] = getpass.getpass(
+                    v = getpass.getpass(
                         f"Please enter the secret_value for the key [{k}]:"
                     )
```

Comment:
Maybe choose whether we're using underscores or hyphens? At moment possibly the inconsistency is confusing?

Reply:
We can probably even remove it and use spaces instead right?